apps/mobile-alphabet/src/app/App.tsx

```diff
@@ -14,6 +15,7 @@ export const App = () => {
             <Stack.Navigator>
                 <Stack.Screen name="Home" component={HomeScreen} />
                 <Stack.Screen name="Menu" component={MenuScreen} />
+                <Stack.Screen name="Detail" component={MenuDetailScreen} />
```

Comment:
Rename the component "AlphabetLetterDetailScreen` or `AlphabetCardDetailScreen`. `MenuDetail` is an oxymoron in my book. The menu is acting as an index component.